Pipfile

```diff
@@ -8,3 +8,4 @@ codechain = {editable = true,path = "."}
 pytest = "*"
 
 [packages]
+dataclasses = {markers = "python_version < '3.7'",version = "*"}
```

Comment:
pipenv don't install `dataclasses; python_version < '3.7'` on `pipenv install --python 3.7 .` if you put it only on `setup.py`.